0.download_data/scripts/nbconverted/download_data.py

```diff
@@ -10,38 +10,57 @@
 import pandas as pd
 
 
-# ### Specify version of mitocheck_data to download from
+# ## Specify mitocheck_data types and paths
+# 
+# We use multiple types of data for model training/testing.
 
 # In[2]:
 
 
-labeled_data_hash = "20369033f579dca1334cb2c58a1c6d532322f93e"
-labeled_data_url = f"https://raw.github.com/WayScience/mitocheck_data/{labeled_data_hash}/3.normalize_data/normalized_data/training_data.csv.gz"
+labeled_data_hash__ic = "20369033f579dca1334cb2c58a1c6d532322f93e"
+labeled_data_url__ic = f"https://raw.github.com/WayScience/mitocheck_data/{labeled_data_hash__ic}/3.normalize_data/normalized_data/training_data.csv.gz"
 
+# ADD PATH TO DATA FROM GITHUB
+# labeled_data_hash = ""
+labeled_data_url__no_ic = f"/home/roshankern/Desktop/Github/mitocheck_data/3.normalize_data/normalized_data__no_ic/training_data.csv.gz"
```

Comment:
is there a place for this data on this repo? It would be great to avoid specific paths that will break on other machines!

Reply:
I wanted to keep it very clear that this path needs to be updated to reflect the path to data on GitHub in `mitocheck_data`. In an effort to not block you, I worked on the repos in the wrong order (`phenotypic_profiling` then `mitocheck_data`). But once https://github.com/WayScience/mitocheck_data/pull/37 is merged we can update the path in this code. Sorry for the confusion!

Reply:
Fixed!